ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Authorizer.scala

```diff
@@ -277,25 +277,32 @@ final class Authorizer(
       authorized: (ClaimSet.Claims, Req) => Either[StatusRuntimeException, Req]
   ): (Req, StreamObserver[Res]) => Unit = (request, observer) => {
     val serverCallStreamObserver = assertServerCall(observer)
-    authenticatedClaimsFromContext()
-      .fold(
-        ex => {
-          observer.onError(ex)
-        },
-        claims =>
-          authorized(claims, request) match {
-            case Right(modifiedRequest) =>
-              call(
-                modifiedRequest,
-                if (claims.expiration.isDefined || claims.resolvedFromUser)
-                  ongoingAuthorization(serverCallStreamObserver, claims)
-                else
-                  serverCallStreamObserver,
-              )
-            case Left(ex) =>
-              observer.onError(ex)
-          },
-      )
+    authenticatedClaimsFromContext() match {
+      case Failure(ex: StatusRuntimeException) => observer.onError(ex)
+      case Failure(ex) =>
+        observer.onError(ex)
+      //        observer.onError(
+      //          LedgerApiErrors.InternalError
+      //            .UnexpectedOrUnknownException(t = ex)(
+      //              new DamlContextualizedErrorLogger(logger, loggingContext, None)
+      //            )
+      //            .asGrpcError
+      //        )
+      case Success(claims) =>
+        (authorized(claims, request): Either[StatusRuntimeException, Req]) match {
+          case Right(modifiedRequest) =>
+            call(
```

Comment:
Here, exceptions thrown from `call` bubble up and are handled (and logged) in the `ErrorInterceptor`